tests/rados/test_mgr_daemon.py

```diff
@@ -0,0 +1,134 @@
+"""
+Module to test MGR functionalities
+
+"""
+
+import time
+
+from ceph.ceph_admin import CephAdmin
+from ceph.rados.core_workflows import RadosOrchestrator
+from ceph.rados.mgr_workflows import MgrWorkflows
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def run(ceph_cluster, **kw):
+    """
+    CEPH manager daemon functional testing
+    Returns:
+        1 -> Fail, 0 -> Pass
+    """
+    log.info(run.__doc__)
+    config = kw["config"]
+    cephadm = CephAdmin(cluster=ceph_cluster, **config)
+    rados_obj = RadosOrchestrator(node=cephadm)
+    mgr_nodes = ceph_cluster.get_nodes(role="mgr")
+    mgr_obj = MgrWorkflows(node=cephadm)
+
+    log.info("Scenario 1: Verification of MGR fail scenario")
+    # Get the active MGR name
+    mgr_stats = mgr_obj.get_mgr_stats()
```

Comment:
With mgr stats, please also verify that there is more than 1 mgr on the cluster. If not, then mgr failover would not work.